src/app.py

```diff
@@ -191,62 +205,76 @@ def update_graph(reports, classes, view, encounter):
                 encounter = encounter
             )
 
-            logs.append(log)
+            if log['entries']:
```

Comment:
maybe better to do
`if log.get('entries', None)`, 
or handle the KeyError works as well